wiki/wiki/doctype/wiki_page/wiki_page.py

```diff
@@ -330,6 +330,7 @@ def get_context(self, context):
 						"url": "/drafts",
 					},
 				],
+				"favicon": wiki_space.favicon or "/assets/frappe/images/frappe-favicon.svg",
```

Comment:
Better way would be to only add favicon to the context if there is a favicon in wiki space, else let the base be used (if we don't add to context). Because we want the app's/website settings favicon to take over and not a hardcoded one like in this case.